src/components/Navbar/MediaLinks.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {
+  FaFacebook,
+  FaTwitter,
+  FaInstagram,
+  FaPinterest,
+} from 'react-icons/fa';
+
+const MediaLinks = () => (
+  <div className="flex justify-center fixed left-6 bottom-16 ">
+    <a
+      href="https://facebook.com"
+      target="_blank"
+      rel="noreferrer"
+      className="mr-3 text-xl sm:text-2xl text-dimIcon hover:text-cyan-800"
+    >
```

Comment:
[OPTIONAL]
These links seem to be similar, can they be abstracted with a component and an array of links maybe to make the code cleaner?